chainregistry.go

```diff
@@ -751,3 +760,43 @@ func initNeutrinoBackend(chainDir string) (*neutrino.ChainService, func(), error
 
 	return neutrinoCS, cleanUp, nil
 }
+
+func parseHeaderStateAssertion(state map[string]string) (
```

Comment:
Any reason we use a map here? While there could be a use for allowing multiple header state assertions, the underlying API doesn't allow doing so.

Reply:
Nit: missing `godoc`.

Reply:
fixed